circle.yml

```diff
@@ -17,6 +17,6 @@ test:
     - docs/dist/pre-commit.sh . || echo "Ignore codesniffer errors"
     # Errors from docs/dist/pre-push.sh are currently ignored
     # because merge commits from github aren't signed
-    - docs/dist/pre-push.sh || echo "Ignore non-signed commits"
+    #- docs/dist/pre-push.sh || echo "Ignore non-signed commits"
```

Comment:
That disables this line, doesn't it? Should we remove it completely together with the preceding comment lines, or do we expect that we'll need that part again in the future?